candle/standard/monadic/holKernelProofScript.sml

```diff
@@ -386,6 +427,45 @@ val term_type = Q.prove(
   imp_res_tac CONTEXT_std_sig >>
   fs[TYPE_def,type_ok_def,is_std_sig_def])
 
+val type_of_has_type = Q.store_thm("type_of_has_type",
```

Comment:
I'm suspicious of this needing an inductive proof... isn't something like it already proved?